src/main/java/command/Command.java

```diff
@@ -0,0 +1,34 @@
+package command;
+
+import henry.TaskList;
+
+/**
+ * The command class is the base for all commands.
+ * It only has one method, execute, which returns a CommandResult.
+ * This method is to be implemented by all subclasses of Command.
+ */
+public abstract class Command {
+
+    protected TaskList taskList;
+
+    /**
+     * Returns a CommandResult object based on the type of Command
+     * executed.
+     *
+     * @return
+     */
+    public abstract CommandResult execute();
+
+    /**
+     * Sets the task list to be used by the command.
+     * The task list is then modified by various commands, such as the {@link TodoCommand}.
+     * <p>
```

Comment:
Unnecessary para tag? 🤔

Reply:
Oops, reviewed this as well! Thanks for pointing it out!